Automattic/jetpack
# A4A Client: Unify disconnect dialog styles

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/246

## Proposed changes:

* Update styles to unify the appearance of the disconnect dialogs used on the plugins screen vs the admin page.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/automattic-for-agencies-dev/issues/246

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






To view the two dialogs:
* Visit the A4A client admin page with a connected site, click the disconnect button.
* Visit the plugins page in wp-admin, click the deactivate button on the A4A client plugin.

Verify that they are visually similar.

## Screenshots

<img width="557" alt="Screenshot 2024-04-23 at 3 44 14 PM" src="https://github.com/Automattic/jetpack/assets/10933065/84958ffa-cbf4-440f-9687-17c3e3f6cbcc">

<img width="568" alt="Screenshot 2024-04-23 at 3 44 20 PM" src="https://github.com/Automattic/jetpack/assets/10933065/6fcd490b-c1e8-4cf0-909b-151230f482b5">
